src/unistd.rs

```diff
@@ -760,11 +760,12 @@ pub fn execvpe(filename: &CString, args: &[CString], env: &[CString]) -> Result<
 ///
 /// This function is similar to `execve`, except that the program to be executed
 /// is referenced as a file descriptor instead of a path.
+// Note for NetBSD and OpenBSD: although rust-lang/libc includes it (under
```

Comment:
Though it's not a blocker for this PR, you should open an issue with libc to get that symbol removed.

Reply:
I've opened https://github.com/rust-lang/libc/pull/1201 to remove `fexecve` from `libc` `netbsdlike`.